ggml/src/ggml-sycl/ggml-sycl.cpp

```diff
@@ -3711,8 +3712,10 @@ static ggml_status ggml_backend_sycl_graph_compute(ggml_backend_t backend, ggml_
         ggml_backend_sycl_graph_compute_impl(sycl_ctx, cgraph);
         model_sycl_graph.end_recording();
 
-        if (!sycl_ctx->exec_graph) {
-            auto exec_graph = model_sycl_graph.finalize({sycl_ex::property::graph::updatable{}});
+        const bool graph_update_support = dpct::get_device(sycl_ctx->device).has(sycl::aspect::ext_oneapi_graph);
```

Comment:
why not update it by `ext_oneapi_limited_graph`?

Reply:
A device with `aspect::ext_oneapi_limited_graph` and not `aspect::ext_oneapi_graph` ` doesn't support update, see https://github.com/intel/llvm/blame/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc#73-querying-device-support.